modules/xmpp/JingleSessionPC.js

```diff
@@ -11,6 +11,13 @@ var XMPPEvents = require("../../service/xmpp/XMPPEvents");
 var RTCBrowserType = require("../RTC/RTCBrowserType");
 var RTC = require("../RTC/RTC");
 
+/**
+ * Constant tells how long we're going to wait for IQ response, before timeout
+ * error is  triggered.
+ * @type {number}
+ */
+const IQ_TIMEOUT = 10000;
```

Comment:
I'm not sure if "const" should work on all the browsers but maybe it's better to change it to "var" for code consistency.